lib/trento/application/projectors/sap_system_projector.ex

```diff
@@ -83,30 +84,95 @@ defmodule Trento.SapSystemProjector do
     end
   )
 
+  project(
+    %ApplicationInstanceHealthChanged{
+      sap_system_id: sap_system_id,
+      host_id: host_id,
+      instance_number: instance_number,
+      health: health
+    },
+    fn multi ->
+      changeset =
+        %ApplicationInstanceReadModel{
+          sap_system_id: sap_system_id,
+          host_id: host_id,
+          instance_number: instance_number
+        }
+        |> ApplicationInstanceReadModel.changeset(%{health: health})
+
+      Ecto.Multi.update(multi, :application_instance, changeset)
+    end
+  )
+
+  @sap_systems_channel "monitoring:sap_systems"
+
   @impl true
   def after_update(
         %SapSystemRegistered{},
         _,
-        %{sap_system: sap_system}
+        %{sap_system: %SapSystemReadModel{} = sap_system}
       ) do
     TrentoWeb.Endpoint.broadcast(
-      "monitoring:sap_systems",
+      @sap_systems_channel,
```

Comment:
nice

Reply:
Yup! Thanks!

Changed also for the database.